client/src/routes/automation/esidom_generator.ts

```diff
@@ -231,6 +233,19 @@ interface BlocksGenerator {
         return JSON.stringify(json);
     };
 
+    block.object_action = (blk: Block) => {
+        const dropdown_entities = blk.getFieldValue('Entities');
+        const dropdown_services = blk.getFieldValue('Services');
+
+        const json: BlocklyJSON = {};
+
+        const entity_id = dropdown_entities.split(':')[1];
```

Comment:
Care, array[1] can return undefined. Try to check for it and do something ? 🤔

Reply:
Check for it then do what ? I don't know.

Reply:
Type is embedded automatically. This should never be undefined unless a code error happend.
Is there a way to send an exception or just ignore this ?
@asoran